news/2021-09-28-project-loved-september-2021.md

```diff
@@ -0,0 +1,457 @@
+
+Don't even consider using mods on it. I'm serious. Anyway, put it simply, this is a map of good quality aimed towards the best of the best. Give it a try, if you dare.
+
+[![](/wiki/shared/news/2021-09-26-project-loved-september-2021/1350854.jpg)](https://osu.ppy.sh/community/forums/topics/1429925)
+
+[schwarzekugel - $trange Attraktor](https://osu.ppy.sh/beatmapsets/1350854#fruits) by [Dika312](https://osu.ppy.sh/users/741613)\
+145 BPM, 2:13 | 4.79★, 4.79★\
+*description written by [Spectator](https://osu.ppy.sh/users/702598)*
+
+Dika312 has recently gained fame among the players for his unique style and stylish patterns. This map, *$trange Attraktor*, shows the excellence of his gimmick mapping skill. The difficulty, which was used in a FreeMod mappool in AxS, is designed to pay attention to both accuracy and precision where you need to treat the slider-ticks as the notes. With NoMod and Hidden, the sliders seem relatively clean due to the use of the most excellent mapping tool of all time, the Droplet Derandomizer by Phob. 
+
+Still, with the addition of HardRock you'll see many wacky slider paths, so you should be prepared for a further accuracy challenge if you ever plan to conquer the map with the HardRock mod.
+
+[![](/wiki/shared/news/2021-09-26-project-loved-september-2021/365254.jpg)](https://osu.ppy.sh/community/forums/topics/1429924)
+
+[goreshit - Satori De Pon!](https://osu.ppy.sh/beatmapsets/365254#fruits) by [TenyO](https://osu.ppy.sh/users/1636577)\
+235 BPM, 2:22 | 9.42★\
+*description written by [Wesley](https://osu.ppy.sh/users/2407265)*
+
+An all-time classic, *Satori De Pon!* by goreshit created by none other than TenyO. You might not know TenyO right off the bat; however, if you have played osu!catch, chances are you have played some of their maps (such as Der Wald and Lone Digger).
+
+Many of TenyO's maps are pretty challenging, and as the difficulty name of this map might suggest, this is no exception. Tenyo's Fruitcore 2015 (CTB Survival) is a map that will test your osu!catch skills to the max, giving you a variety of patterns ranging from jumps, hyperdash chains, zig-zags, and of course, an all-time favourite, pixels! Now, the hp drain is relatively low, but do not let this fool you. If you miss one of the more packed sections of the map, you'll quickly see that the hp drain feels much higher.
+
+[![](/wiki/shared/news/2021-09-26-project-loved-september-2021/832873.jpg)](https://osu.ppy.sh/community/forums/topics/1429923)
+
+[League of Legends - Warriors (feat. Imagine Dragons)](https://osu.ppy.sh/beatmapsets/832873#fruits) by [Minato Yukina](https://osu.ppy.sh/users/531253)\
```

Comment:
i'm gonna go by the assumption that this change was intentional (instead of `Imagine Dragons - Warriors` as on the website currently)